spring-app/src/main/java/pl/edu/agh/cqm/data/dto/ConfigParametersDTO.java

```diff
@@ -0,0 +1,18 @@
+package pl.edu.agh.cqm.data.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class ConfigParametersDTO {
+
+    private List<String> cdns;
+
+    private int activeSamplingRate;
+    private int activeTestIntensity;
+
+    private int passiveSamplingRate;
```

Comment:
Let's add some very basic validation for those fields, at least `@NotNull` and `@Min(1)`